src/main/resources/bank/items/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ignore everything except .gitignore
```

Comment:
That's certainly an option, but I added this .gitignore at this level to force git to add the "empty" directory.